### PR TITLE
Collect changelogs from all branches in main

### PR DESCRIPTION
**Pull request by mdellweg**
[noissue]

Review Checklist:
- [ ] An issue is properly linked. [feature and bugfix only]
- [ ] Tests are present or not feasible.
- [ ] Commits are split in a logical way (not historically).

### CODE REVIEW

File: .github/workflows/changes.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Collect changes
+        run: |
+          pip install GitPython
+          python3 .ci/scripts/collect_changes.py
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          title: "Update Changelog"
+          branch: "update_changes"
+          delete-branch: true
```
Comment (gerrod3) [inline .github/workflows/changes.yml:30]: I think you need to specify the commit message to have [noissue]. Also, do you need to pass in the token for the pull request to be made?

Comment (mdellweg) [reply .github/workflows/changes.yml:30]: Hard to say without trying...

Comment (mdellweg) [reply .github/workflows/changes.yml:30]: [noissue] is in the commit created by the collect script. This workflow is not supposed to add another commit.

Comment (mdellweg) [reply .github/workflows/changes.yml:30]: I was tempted to just push to main from the workflow...

**Review by gerrod3 — APPROVED** [review]

**Pull request merged**